sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/converter/AccessTokenScopesIT.java

```diff
@@ -18,29 +18,33 @@
 
 import java.util.*;
 
-public class RefreshTokenScopesIT {
+public class AccessTokenScopesIT {
 
     @Test
     public void testRefreshTokenConverter() {
         try (AppRunner app = new AppRunner(DumbApp.class)) {
             SeleniumTestUtils.addProperty(app);
-            app.property("azure.activedirectory.authorization.office.scopes", "https://manage.office.com/ActivityFeed.Read");
-            app.property("azure.activedirectory.authorization.graph.scopes", "https://graph.microsoft.com/User.Read");
+            app.property("azure.activedirectory.authorization.office.scopes", "https://manage.office.com/ActivityFeed.Read , https://manage.office.com/ActivityFeed.ReadDlp , https://manage.office.com/ServiceHealth.Read");
+            app.property("azure.activedirectory.authorization.graph.scopes", "https://graph.microsoft.com/User.Read , https://graph.microsoft.com/AccessReview.Read.All");
             List<String> endPoints = new ArrayList<>();
-            endPoints.add("api/office");
-            endPoints.add("api/azure");
-            endPoints.add("api/graph");
-            endPoints.add("api/arm");
+            endPoints.add("accessTokenScopes/office");
+            endPoints.add("accessTokenScopes/azure");
+            endPoints.add("accessTokenScopes/graph");
+            endPoints.add("accessTokenScopes/arm");
             Map<String, String> result = SeleniumTestUtils.get(app, endPoints);
 
-            Assert.assertFalse(result.get("api/office").contains("profile"));
-            Assert.assertTrue(result.get("api/office").contains("https://manage.office.com/ActivityFeed.Read"));
+            Assert.assertFalse(result.get("accessTokenScopes/office").contains("profile"));
+            Assert.assertTrue(result.get("accessTokenScopes/office").contains("https://manage.office.com/ActivityFeed.Read"));
+            Assert.assertTrue(result.get("accessTokenScopes/office").contains("https://manage.office.com/ActivityFeed.ReadDlp"));
+            Assert.assertTrue(result.get("accessTokenScopes/office").contains("https://manage.office.com/ServiceHealth.Read"));
 
-            Assert.assertTrue(result.get("api/azure").contains("profile"));
-            Assert.assertTrue(result.get("api/azure").contains("https://graph.microsoft.com/User.Read"));
+            Assert.assertTrue(result.get("accessTokenScopes/azure").contains("profile"));
+            Assert.assertTrue(result.get("accessTokenScopes/azure").contains("https://graph.microsoft.com/AccessReview.Read.All"));
+            Assert.assertTrue(result.get("accessTokenScopes/azure").contains("https://graph.microsoft.com/User.Read"));
```

Comment:
So azure's scope is always same to graph's?

Reply:
The scopes of each client of the same resource is the same